uber/site_sections/preregistration.py

```diff
@@ -254,9 +244,6 @@ def prereg_payment(self, session, payment_id, stripeToken):
                 if attendee.amount_extra:
                     attendee.amount_paid = attendee.total_cost
             session.add(group)
```

Comment:
The banned email sending was moved to the attendee presave adjustment, so it's not routing through the group anymore. Or do we need the group.leader here for something else?